src/components/molecules/UiHeader/UiHeader.vue

```diff
@@ -189,24 +189,24 @@ onUnmounted(() => {
 
 <style lang="scss">
 @use "../../../styles/functions";
+@use "../../../styles/mixins";
 
 .ui-header {
   $element: header;
 
+  --navigation-justify-content: flex-end;
+
   background: functions.var($element, background, var(--color-background-brand));
 
   &__header {
+    @include mixins.use-logical($element, padding, var(--space-20));
+
     display: flex;
     width: 100%;
     max-width: functions.var($element, max-width, 61.25rem);
     align-items: center;
     justify-content: space-between;
-    padding: functions.var($element, padding, 0 var(--space-20));
-    margin: 0 auto;
```

Comment:
this `margin` is important because is center header content with some `max-width`
on this PR
![Zrzut ekranu 2022-12-7 o 13 45 37](https://user-images.githubusercontent.com/12138170/206182917-df01b64c-0ed3-4445-9a79-ad5e2a4cd9b5.png)
expected
![Zrzut ekranu 2022-12-7 o 13 45 46](https://user-images.githubusercontent.com/12138170/206182954-f6a54ea1-4e35-4da0-a1de-dfe5dee376f8.png)